hicommonwealth/commonwealth
# Implement twitter embed

## Link to Issue
Closes: #3551

## Description of Changes
- Implements twitter embeds so that twitter links are converted to embeds on the fly

## Test Plan
- Open any quill editor in richtext mode
- Paste the following link: `https://twitter.com/sketch/status/1017789080871030784`
- Confirm that the tweet from the Sketch account shows in the editor
- Open the preview modal– confirm that the preview renders the tweet embed correctly
- Post the comment/thread– confirm that the tweet embed renders in final view area

## Deployment Plan
N/A

## Other Considerations
N/A
